test/base/src/androidTest/kotlin/io/realm/test/shared/QueryTests.kt

```diff
@@ -689,22 +942,109 @@ class QueryTests {
 
     @Test
     @Ignore
-    fun sum_generic_asFlow_throwsIfInvalidType() {
+    fun sum_asFlow_throwsIfInvalidType() {
         // TODO
     }
 
     @Test
     @Ignore
-    fun sum_generic_asFlow_throwsIfInvalidProperty() {
+    fun sum_asFlow_throwsIfInvalidProperty() {
         // TODO
     }
 
     @Test
     @Ignore
-    fun max_find() {
+    fun sum_asFlow_throwsIfRealmInstant() {
         // TODO
     }
 
+    @Test
```

Comment:
Don't know if there is infrastructure to collect multiple errors in a test, but would maybe make sense to do an grouping of tests with something like:
```
@Test
fun aggregators() {
    for (aggregator in setOf(MIN, MAX, COUNT, ...) {
        find_emptyTable(...)
        find_allNullValues(....)
       ...
    }
}
```
To ensure that we catch all the uses cases for each aggregator type. Don't know if the implementations vary too much for it to make sense though.

Reply:
I see. I guess we could then use the pattern with a `AggregatorTest` interface with the specific tests and have a test class for each aggregator. This haven't worked well on the `NotificationTests` though. Part of the issues with that is:
- that default methods in the interface will trigger running the actual test on the interface (probably the underlying `DefaultImpl` class) but the test executor fails
- running the `jvmTest`

I previously tried summarizing the issues in https://github.com/realm/realm-kotlin/issues/397. Some of the it could be avoided by eliminating default implementations and gradle/test-runner infrastructure might have changes, so maybe worth a very quick shot again 🤔